### DISCUSSION TITLE
osu!lazer lack of sound in replay if using a specific audio offset

### DISCUSSION THREAD

**Discussion by supahero1** [ppy/osu#31343] [Q&A]
### Type

Game behaviour

### Bug description

Basically: go in a map with autoplay or a replay so that you can move around the timeline. Then, near the beginning, use arrow left to rewind the playback. When beatmap audio offset is on, you can make lazer not play the beatmap music at all for the entire duration of the replay and afterwards. This is very easy to consistently replicate at higher speeds - that's why in the video I'm adding DT, after which you can see I'm able to invoke this behavior 4 times in a row whereas without DT I could only do it once in like 6 attempts. My global audio offset is 0.

For this map specifically, 7.4-7.5ms offset seems to be the magic offset (I tested these 2 values, something near them probably will work too). I could not replicate this behavior with any other randomly chosen offset with DT, nor any other map, and honestly I don't want to waste too much time trying.

### Screenshots or videos

https://streamable.com/3jhx5i

### Version

2024.1224.1-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18269549/compressed-logs.zip)

**Comment by bdach** [comment]
Cannot reproduce using provided information. Wouldn't be surprised if this was a special snowflake linux issue.
